src/SIL.XForge.Identity/SIL.XForge.Identity.csproj

```diff
@@ -11,6 +11,7 @@
     <PackageReference Include="Microsoft.AspNetCore.Mvc" Version="2.1.2" />
     <PackageReference Include="Microsoft.AspNetCore.StaticFiles" Version="2.1.1" />
     <PackageReference Include="Microsoft.Extensions.FileProviders.Embedded" Version="2.1.1" />
+    <PackageReference Include="NETCore.MailKit" Version="2.0.2" />
```

Comment:
I don't think you need this anymore, since you moved the send email code to `SIL.XForge`.